src/domain/thirdpartyRequests/transactions.ts

```diff
@@ -0,0 +1,177 @@
+ Those individuals should have their names indented and be marked with a '-'. Email address can be
+ added optionally within square brackets <email>.
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+
+ - Sridhar Voruganti <sridhar.voruganti@modusbox.com>
+
+ --------------
+ ******/
+'use strict'
+
+import Hapi from '@hapi/hapi'
+import Logger from '@mojaloop/central-services-logger'
+import { Factory, ReformatFSPIOPError } from '@mojaloop/central-services-error-handling'
+import { Enum, Util } from '@mojaloop/central-services-shared'
+import { EventStateMetadata, EventStatusType } from '@mojaloop/event-sdk'
+import Mustache from 'mustache'
+import Config from '~/shared/config'
+import inspect from '~/shared/inspect'
+import { getStackOrInspect } from '~/shared/util'
+import * as types from '~/interface/types'
+
+/**
+ * Forwards POST transactions requests to destination FSP for processing
+ * @param {string} path Callback endpoint path
+ * @param {object} headers Headers object of the request 
+ * @param {string} method The http method POST
+ * @param {object} params Params object of the request 
+ * @param {object} payload Body of the POST request
+ * @param {object} span request span
+ * @throws {FSPIOPError} Will throw an error if no endpoint to forward the transactions requests is
+ * found, if there are network errors or if there is a bad response
+ * @returns {Promise<void>}
+ */
+async function forwardTransactionRequest(path: string, headers: Hapi.Util.Dictionary<string>, method: string, params: Hapi.Util.Dictionary<string>, payload?: types.TThirdPartyTransactionRequest, span?: any): Promise<void> {
+
+  const childSpan = span ? span.getChild('forwardTransactionRequest') : undefined
+  const fspiopSource: string = headers[Enum.Http.Headers.FSPIOP.SOURCE]
+  const fspiopDest: string = headers[Enum.Http.Headers.FSPIOP.DESTINATION]
+  const payloadLocal = payload || { transactionRequestId: params.ID }
+  const transactionRequestId: string = (payload && payload.transactionRequestId) || params.ID
+  const endpointType = Enum.EndPoints.FspEndpointTypes.THIRDPARTY_CALLBACK_URL_TRX_REQ_POST
+  let fspiopError: typeof Factory.FSPIOPError | undefined
+  let endpoint: string | undefined
+
+  try {
+    endpoint = await Util.Endpoints.getEndpoint(
+      Config.SWITCH_ENDPOINT,
+      fspiopDest,
+      endpointType)
+    Logger.info(`Resolved PAYER party ${endpointType} endpoint for transactionRequest
+     ${transactionRequestId} to: ${inspect(endpoint)}`)
+    const fullUrl: string = Mustache.render(endpoint + path, { ID: transactionRequestId })
+    Logger.info(`Forwarding transaction request to endpoint: ${fullUrl}`)
+
+    await Util.Request.sendRequest(
+      fullUrl,
+      headers,
+      fspiopSource,
+      fspiopDest,
+      method,
+      method.toUpperCase() !== Enum.Http.RestMethods.GET ? payloadLocal : undefined,
+      Enum.Http.ResponseTypes.JSON,
+      childSpan)
```

Comment:
is it intentionally to not to use jwsSigner?

Reply:
@lewisdaly && @elnyry-sam-k - any thoughts?

Reply:
I think that's because we are just passing on the request as is from the source DFSP, which already will have the signed headers. The switch doesn't need to re-sign the request. I could be wrong however.